assets/js/ludo.js

```diff
@@ -0,0 +1,298 @@
+var currPos = 0;
+var step = 49.5;
+var currcolor = "";
+var NumOfPaw = "";
+var num = 0;
+var clicked = false;
+var currpawn = "";
+var allcolor = ["red", "blue", "green", "yellow"];
+var pawnOut = { red: 0, blue: 0, green: 0, yellow: 0 }
+function HaveHover() {
+    var count = 0;
+    var toKill = "";
+    for (var i = 0; i < allcolor.length; i++) {
+        for (var n = 1; n <= 4; n++) {
+            var firstPawn = document.getElementById(allcolor[i] + "pawn" + n);
+            var secondPawn = document.getElementById(currpawn);
+            if (firstPawn.style.top == secondPawn.style.top && firstPawn.style.left == secondPawn.style.left && currcolor != allcolor[i] && currPos + num < 44) {
+                count++;
+                toKill = allcolor[i] + "pawn" + n;
+                return toKill;
+            }
+        }
+    }
+    return false;
+}
+function Stuck() {
+    var text = document.getElementById('player');
+    if (onboard[currpawn] == 0 || currPos + num > 44) {
+        if (DontHaveOtherFree() || currPos + num > 44) {
+            var badtext = document.getElementById('badtext');
+            badtext.innerText = "Unfortunatlly you stuck";
+            clicked = false;
+            var dice = document.getElementById('dice');
+            dice.style.backgroundImage = "url(../assets/Images/ludo/dice.gif)";
```

Comment:
@991rajat  mostly all the links from html file are working fine, I guess the problem is with the path URL present in js file
can you review this path URLs because these are working fine in my machine